src/landing/model_files_item.rs

```diff
@@ -318,19 +318,14 @@ impl Widget for ModelFilesItem {
     }
 }
 
-impl WidgetMatchEvent for ModelFilesItem {
-    fn handle_actions(&mut self, cx: &mut Cx, actions: &Actions, scope: &mut Scope) {
-        let widget_uid = self.widget_uid();
+impl MatchEvent for ModelFilesItem {
+    fn handle_actions(&mut self, cx: &mut Cx, actions: &Actions) {
```

Comment:
Remember to keep this one as `WidgetMatchEvent` instead of switching to `MatchEvent`.
I've learned after we talked in person that MatchEvent was made specifically for the main app module.

Reply:
ok，I got it.Thanks for your reviewing.

Reply:
@Guocork remember this one, I think you haven't updated it.
After that we're good to go!